content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1004,13 +1004,10 @@ observe that pods from a Job are stuck with the tracking finalizer.
 
 ### Elastic Indexed Jobs
 
-{{< feature-state for_k8s_version="v1.27" state="beta" >}}
+{{< feature-state feature_gate_name="ElasticIndexedJob" >}}
```

Comment:
```suggestion
{{< feature-state for_k8s_version="v1.31" state="stable" feature_gate_name="ElasticIndexedJob" >}}
```

Reply:
The existing content `{{< feature-state feature_gate_name="ElasticIndexedJob" >}}` is accurate. We can check the [Build Preview link (here)](https://deploy-preview-46991--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/job/#elastic-indexed-jobs) to see how it renders. 

For information on using the `feature-state` shortcode, refer to the [documentation (here)](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#feature-state-retrieval-from-description-file).

Reply:
Oh I see, so it pulls the information from the other file. Neat :)